Project.toml

```diff
@@ -21,11 +21,11 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 CSV = "0.5"
-FillArrays = "0.7"
-Optim = "0.19"
+FillArrays = "0.6, 0.7"
+Optim = "0.18, 0.19"
 Parameters = "0.10"
-RDatasets = "0.6"
-StatsBase = "0.32"
+RDatasets = "0.6.3, 0.6.4"
```

Comment:
`RDatasets = "0.6.3"` means `[0.6.3, 0.7)` so the `0.6.4` is redundant.

Reply:
ah - OK